domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewService.kt

```diff
@@ -31,4 +34,71 @@ class ReviewService(
    */
   fun getCompletedReviewsForPrisoner(prisonNumber: String): List<CompletedReview> =
     reviewPersistenceAdapter.getCompletedReviews(prisonNumber)
+
```

Comment:
My basic thought process is that client code (which I have in another branch) that needs to either create or re-calculate someone's Review will first call `determineReviewScheduleCalculationRule` to get the calculation rule; and then will call `calculateReviewWindow` to get the review window ("from" date and "to" date)